.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -9,11 +9,15 @@ assignees: ''
 
```

Comment:
at the top
```suggestion
<!--anything that looks like this is a comment and cannot be seen after creating the issue.-->
```

Reply:
I find this actually quite annoying, but I know what they are....